tools/kvrocks2redis/parser.cc

```diff
@@ -155,7 +155,7 @@ rocksdb::Status WriteBatchExtractor::PutCF(uint32_t column_family_id, const Slic
   std::string ns, user_key, sub_key;
   std::vector<std::string> command_args;
   if (column_family_id == kColumnFamilyIDMetadata) {
-    ExtractNamespaceKey(key, &ns, &user_key);
+    ExtractNamespaceKey(key, &ns, &user_key, this->is_slotid_encoded_);
```

Comment:
other private parameter (first_seen_, aof_strings_, etc..) did not use this->,  is this on purpose ?

Reply:
Good, you're right that we shouldn't use `this` keyword